TestFoundation/Utilities.swift

```diff
@@ -497,6 +497,17 @@ func shouldAttemptXFailTests(_ reason: String) -> Bool {
     }
 }
 
+func shouldAttemptWindowsXFailTests(_ reason: String) -> Bool {
+  var isOSWindows: Bool = false
+#if os(Windows)
+  isOSWindows = true
+#endif
+
+  if isOSWindows && shouldRunXFailTests { return true }
+  try? FileHandle.standardError.write(contentsOf: Data("warning: Skipping test expected to fail with reason '\(reason)'\n".utf8))
+  return false
```

Comment:
This is disabling the test on non-Windows afaict.

Reply:
Maybe `if isOSWindows { try? FileHandle…; return false } else { return shouldAttemptXFailTests(reason) }`

Reply:
err, right

Reply:
Actually, what I screwed up was my boolean - I indented to use `if !isOSWindows || shouldRunXFailTests`.  I are dumb, nothing new here.